app/models/appointment_reminder.rb

```diff
@@ -0,0 +1,11 @@
+class AppointmentReminder < ActiveRecord::Base
```

Comment:
[non blocking] I presume we have discussed why this can't be a `Communication`. Can someone please point me to it?

Reply:
Asking because we originally considered appointment reminder as the name for this sort of thing, and then picked Communication from FHIR since it was pertinent.

Reply:
No one has talked to me about what a Communication is at all. I'm aware of the model and it will be involved in this epic, but it was going to be used in the same way that it currently is. AppointmentReminder was going to essentially take the `Appointment#remind_on` functionality, but I wasn't planning to change Communication significantly. Let's discuss this today. I would love to use fewer models if viable, but Communication's current schema doesn't seem viable for this because it lacks remind_on and message attributes.